README.md

```diff
@@ -90,6 +88,44 @@ final class LastPosts extends Widget
 }
 ```
 
+### LocaleState of `View` and `WebView` services
+`LocaleState` like the previous state is not cloned when services are cloned. You can change the locale by using `setLanguage()`, which will be applied to all other instances including existing ones. If you need to change the locale only for a single instance, you can use the immutable `withLanguage()` method. Locale will be applied to all views rendered within views with `render()` calls.
+
+example with mutable method:
+```php
+final class LocaleMiddleware implements MiddlewareInterface
+{    
+    ...
+    private WebView $view;
+    ...
+    public function __construct (
+        ...
+        WebView $view
+        ...
+    ) {
+        $this->view = $view;
+    }
+    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
+    {
+        ...
+        $this->view->setLanguage($locale);
+        ...
+    }
+}
+````
+or immutable method:
+```php
+final class BlogController {
+    private WebView $view;
+    public function __construct (WebView $view) {
+        $this->view = $view;
+    }
+    public function index() {
+        return $this->view->withLanguage('es')->render('index');
+    }
+}
+```
+
 To get a deep cloned `View` or `WebView` use `withClearedState()`: 
```

Comment:
Deep clone should be before locale state part.

Reply:
Why?, deep clone also resets locale state.

Reply:
Then need heading. Now seems that deep clone into "Locale state".